src/App.js

```diff
@@ -1,42 +1,24 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Header } from "./component";
+import { PrivateRoute } from "./PrivateRoute";
+import { Routes, Route } from "react-router-dom";
+import { HomePage, LandingPage, Signin, Signup } from "./pages";
+import Mockman from "mockman-js";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
-    </div>
+    <>
+      <Header />
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="/signin" element={<Signin />} />
+        <Route path="/signup" element={<Signup />} />
+        <Route element={<PrivateRoute />}>
+          <Route path="/homepage" element={<HomePage />} />

```

Comment:
why you have hompage and landing page, is it in prd to have these separate pages?
 why m saying because as reviewer i think  the purpose of both pages is same

Reply:
Homepage will be displayed after login  where all the notes will be displayed  if it creates confusion for reviewer try to define with different name